sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_client.py

```diff
@@ -11,7 +11,7 @@
 from azure.core.credentials import AzureKeyCredential
 from azure.core.pipeline.policies import AzureKeyCredentialPolicy
 from ._generated import BatchDocumentTranslationClient as _BatchDocumentTranslationClient
-from ._generated.models import BatchStatusDetail as _BatchStatusDetail
+from ._generated.models import TranslationStatus as _TranslationStatus
```

Comment:
this is because of the swagger update, right? did you change any public api ? not sure how I can see that :(

Reply:
This is updated due to the renames in the swagger. No public API changes.